pylint/lint/expand_modules.py

```diff
@@ -1,9 +1,28 @@
 import os
 import sys
-from typing import List, Pattern, Tuple
+from typing import List, Pattern, Tuple, Union
 
 from astroid import modutils
 
+if sys.version_info >= (3, 8):
+    from typing import Literal, TypedDict
+else:
+    from typing_extensions import Literal, TypedDict
+
+
+class MODULE_DESCRIPTION_DICT(TypedDict):
```

Comment:
After a (potential) merge of #4973 these should be added to `pylint/typing`.
This PR is therefore blocked by #4973